plugins/header_rewrite/operators.cc

```diff
@@ -610,21 +589,9 @@ OperatorSetRedirect::exec(const Resources &res) const
       const_cast<Resources &>(res).changed_url = true;
       res._rri->redirect                       = 1;
     } else {
+      Dbg(pi_dbg_ctl, "OperatorSetRedirect::exec() hook=%d", int(get_hook()));
```

Comment:
is logging the int meaningful? Would it be more readable to have the string?

Reply:
I didn't have any trouble figuring it out, from looking at the code:  https://github.com/apache/trafficserver/blob/aaff5a9c922f4fbf1c3eacbf5864491d9cdd30c8/include/ts/apidefs.h.in#L450